R/CreatePackageReport.R

```diff
@@ -85,7 +85,10 @@ CreatePackageReport <- function(pkg_name
     
     silence_logger()
     rmarkdown::render(
-        system.file(file.path("package_report", "package_report.Rmd"), package = "pkgnet")
+        system.file(file.path("package_report", "package_report.Rmd")
+                    , package = "pkgnet"
+                    , lib.loc = .libPaths()[1]
```

Comment:
I'm wary about hardcoding in `.libPaths()[1]` in these various places. What if somebody installed their packages in a different library tree? Or if somebody installed their packages, and then later on added a new library path to the beginning of their `.libPaths()` vector?